spec/services/applications/accept_spec.rb

```diff
@@ -251,7 +251,7 @@
     end
 
     describe "NPQ capping" do
-      let(:cohort) { create(:cohort, :current, :with_funding_cap) }
+      let(:cohort) { create(:cohort, :next, :with_funding_cap) }
```

Comment:
I think it's ok to keep to current for this and below as we are just testing and prefer not to test for next as that will always change